nautobot/dcim/templates/dcim/interface.html

```diff
@@ -223,22 +218,9 @@
                     </table>
                 </div>
             {% endif %}
-            {% plugin_right_page object %}
-        </div>
-    </div>
-    <div class="row">
-        <div class="col-md-12">
-            {% include 'panel_table.html' with table=ipaddress_table heading="IP Addresses" %}
-        </div>
-    </div>
-    <div class="row">
-        <div class="col-md-12">
-            {% include 'panel_table.html' with table=vlan_table heading="VLANs" %}
-        </div>
-    </div>
-    <div class="row">
-        <div class="col-md-12">
-            {% plugin_full_width_page object %}
-        </div>
-    </div>
-{% endblock %}
+{% endblock content_right_page %}
+
+{% block content_full_width_page %}
+    {% include 'panel_table.html' with table=ipaddress_table heading="IP Addresses" %}
+    {% include 'panel_table.html' with table=vlan_table heading="VLANs" %}
```

Comment:
Just checking, as I see these were each in separate row/column wrappers before - does this render correctly without those?

Reply:
Yes, it renders correctly, because each `panel_table.html` include are inside of a div:

```jinja2
<div class="panel panel-{{ panel_class|default:'default' }}">
...
</div>
```
(The "row" div was extraneous for these since they automatically fill the containing box and display as block by default.)